### PR TITLE
capitalization fixes

### PR DESCRIPTION
**Pull request by Mqisty**
fixed dumb shit that was bothering me for no reason, not sure why it says i edited line 642

also i could do this everywhere like I did with micros repos if you want im bored af

### CODE REVIEW

File: files/botautoresponse.json [hunk 1]
```diff
@@ -648,7 +648,7 @@
                 ".null"
             ]
         ],
-        "response": "To fix the problem:\n1. Go to <https://account.xbox.com/en-us/settings>\n2. Navigate to the `Xbox One/Windows 10 Online Safety` tab. If you are under 18, you may need a parent or guardian to help by getting them to create a family and add you to it.\n3. From there, they should be able to enable multiplayer for your account.\n4. Relaunch Minecraft and you should be able to join any multiplayer server."
+        "response": "To fix the problem:\n1. Go to <https://account.xbox.com/en-us/settings>\n2. Navigate to the `Xbox One/Windows 10 Online Safety` tab. If you are under 18, you may need a parent or guardian to help by getting them to create a family and adding you to it.\n3. From there, they should be able to enable multiplayer for your account.\n4. Relaunch Minecraft and you should be able to join any multiplayer server."
```
Comment (KTibow) [inline files/botautoresponse.json]: this feels wrong both ways

File: files/botautoresponse.json [hunk 2]
```diff
@@ -657,7 +657,7 @@
                 ".onecore"
             ]
         ],
-        "response": "If you\u2019re getting the \u201cW-CORE or OneCore has failed to download message\u201d, please send your `latest.log` and confirm you can visit the following websites: https://repo.woverflow.cc/ and https://woverflow.cc/ (make sure you\u2019re not using a VPN and your Antivirus isn\u2019t blocking it)"
+        "response": "If you\u2019re getting the \u201cW-CORE or OneCore has failed to download message\u201d, please send your `latest.log` and confirm you can visit the following websites: https://repo.woverflow.cc/ and https://woverflow.cc/ (make sure you\u2019re not using a VPN and your AntiVirus isn\u2019t blocking it)"
```
Comment (KTibow) [inline files/botautoresponse.json]: Antivirus is not enough. You need AntiVirus, world's one and only antivirus with data recovery software. Think beyond Antivirus, think AntiVirus.

File: files/botautoresponse.json [hunk 3]
```diff
@@ -762,4 +762,4 @@
         ],
         "response": "https://maven.minecraftforge.net/net/minecraftforge/forge/1.8.9-11.15.1.2318-1.8.9/forge-1.8.9-11.15.1.2318-1.8.9-installer.jar"
     }
-]
+]
```
Comment (KTibow) [inline files/botautoresponse.json]: :suspect: 
please always add new lines at the end of files

File: files/botautoresponse.json (hunk 1 above)
Comment (Mqisty) [reply files/botautoresponse.json]: yeah i didnt know what it meant so i just left it i agree tho

File: files/botautoresponse.json (hunk 2 above)
Comment (KTibow) [reply files/botautoresponse.json]: Make sure you're not using a VPN, your internet isn't restricted, and your antivirus isn't blocking it

Comment (Mqisty) [reply files/botautoresponse.json]: we dont talk about it

File: files/botautoresponse.json (hunk 3 above)
Comment (Mqisty) [reply files/botautoresponse.json]: idk what happened there it says that i edited lines that i didnt edit

Comment (KTibow) [reply files/botautoresponse.json]: image

File: files/botautoresponse.json (hunk 2 above)
Comment (KTibow) [reply files/botautoresponse.json]: Antivirus is not enough. You need Antivirus, world's one and only antivirus with data recovery software. Think beyond antivirus, think Antivirus.

File: files/botautoresponse.json (hunk 3 above)
Comment (Mqisty) [reply files/botautoresponse.json]: my bad it does that when i save the file for some reason, prettier probably

Comment (KTibow) [reply files/botautoresponse.json]: Prettier adds lines at the end of files IIRC, check your global config

Comment (Mqisty) [reply files/botautoresponse.json]: you were right, prettier was disabled for some reason idek anymore my brain is going to explode

**Review by KTibow — APPROVED** [review]
LGTM!

**Comment by nacrt** [comment]
is this pr done?

**Comment by Mqisty** [comment]
if everything looks good to you yeah

**Pull request merged**